source/future.tex

```diff
@@ -221,34 +221,28 @@
 
 \pnum
 The header \libheaderref{stdalign.h} has the following macros:
-\indexlibraryglobal{__alignas_is_defined}%
 \begin{codeblock}
-#define @\xname{alignas_is_defined}@ 1
-#define @\xname{alignof_is_defined}@ 1
+#define @\libglobal{alignas_is_defined}@ 1
```

Comment:
This drop the double-underscore prefix  Did you actually run "diffpdf" on the resulting, rendered document?

Reply:
Verified with diffpdf before committing the latest update.